src/cli/commands/version.js

```diff
@@ -130,6 +131,14 @@ export async function setVersion(
     return () => Promise.resolve();
   }
 
+  function buildCommitArgs(args): Array<string> {
```

Comment:
I don't think abstracting this into a separate function adds much value unless you use it in a unit test.